xml_inscriptions/bib_only/MA.Bos.MFA.G.84.48.xml

```diff
@@ -62,17 +125,49 @@
       <revisionDesc>
          <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009 EpiDoc</change>
          <change when="2011-05-13" who="Elli Mylonas">More tweaks for bib only inscriptions</change>
+         <change when="2023-07-18" who="Jean Wanlass">added metadata, transcription</change>
       </revisionDesc>
    </teiHeader>
-   <!--facsimile element goes here-->
+   <!--  *************************************  <facsimile>  *********************************  -->
+   <!-- Used to record images of an inscription.   -->
+   
+   <facsimile>
+      <surface>
+         <desc/>
+         <graphic url="MA.Bos.MFA.G.84.48.jpg"/>
+      </surface>
+   </facsimile>
+   
    <text>
       <body>
          <div type="edition">
-         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
-         </div>
+            <ab><gap extent="unknown" unit="character" reason="illegible"></gap>εξοδ<gap extent="unknown" unit="character" reason="illegible"></gap> <lb/>
+               
+               <gap extent="unknown" unit="character" reason="illegible"></gap><unclear>  γ̣</unclear>αριστω Βάκχω <gap extent="unknown" unit="character" reason="illegible"></gap>  <lb/>  
```

Comment:
Something has gone awry with what you have marked `<unclear>` here. Also if you are marking something as such, you don't need to include the underdot in your text.